esp/clients/roxiecontrol.cpp

```diff
@@ -20,10 +20,13 @@
 
 const unsigned roxieQueryRoxieTimeOut = 60000;
 
-void checkRoxieControlExceptions(IPropertyTree *msg)
+static void checkRoxieControlExceptionsEx(IPropertyTree *response, const char *traceInfoRequest, unsigned traceInfoWait) //there are two copies of this function right now, renaming this one 'Ex' for now
```

Comment:
The other copy seems to be a member function, so shouldn't clash, but making this static is a good move.  Should the other function be changed in the same way?

Reply:
Agreed, I will update the other as well.